packages/bs-ethereum/src/services/wallet-connect/WalletConnectServiceEthereum.ts

```diff
@@ -40,32 +40,39 @@ export class WalletConnectServiceEthereum<N extends string, A extends TBSNetwork
     'wallet_addEthereumChain',
   ]
 
-  readonly #service: IBSEthereum<N, A>
+  protected readonly _service: IBSEthereum<N, A>
 
   constructor(service: IBSEthereum<N, A>) {
-    this.#service = service
-    this.chain = `${this.namespace}:${this.#service.network.id.toString()}`
+    this._service = service
+    this.chain = `${this.namespace}:${this._service.network.id.toString()}`
   }
 
   protected async _resolveParams(args: TWalletConnectServiceRequestMethodParams<N>) {
```

Comment:
Did you test WalletConnect with other EVMs? I’m asking because these changes impact all Ethereum-based services.